Widgets/openGL_widgets/ArrowGLContext.py

```diff
@@ -26,7 +27,7 @@ def __init__(self, data_dict, parent):
         self.ZERO_ROT = np.array([[1, 0, 0],
                                   [0, 1, 0],
                                   [0, 0, 1]])
-        self.SIDES = 32
+        self.SIDES = self.decimate
```

Comment:
What reason is behind creating another variable? Can't we just use self.decimate instead?

Reply:
this variable changes context depending on the class thats why it is assigned to class-specific variable